src/components/SingleVideo.vue

```diff
@@ -3,87 +3,38 @@
   Single video component
   todo: fill out this section
   ***************************/
+
+  import CardFooter from './CardFooter.vue'
```

Comment:
I'm just gonna merge without actually looking at what this looks like. my hunch is it will look a little funny within the video module. maybe I'm wrong though

Reply:
You're right, it kinda does. it's kinda just a placeholder for any sort of actions. it could be contextual to type of card.